csv/src/main/scala/akka/stream/alpakka/csv/CsvToMapStage.scala

```diff
@@ -12,15 +12,16 @@ import akka.util.ByteString
 import scala.collection.immutable
 
 /**
- * Converts incoming List[ByteString] to Map[String, ByteString].
+ * Scala API: Converts incoming [[List[ByteString]]] to [[Map[String, ByteString]]].
+ * @see akka.stream.alpakka.csv.CsvToMapJavaStage
  *
  * @param columnNames If given, these names are used as map keys; if not first stream element is used
  * @param charset Character set used to convert header line ByteString to String
  */
-class CsvToMapStage(columnNames: Option[immutable.Seq[String]], charset: Charset)
-    extends GraphStage[FlowShape[immutable.List[ByteString], Map[String, ByteString]]] {
+class CsvToMapStage(columnNames: Option[List[String]], charset: Charset)
```

Comment:
isn't immutable `Seq` better here, to give more flexibility to what can be accepted ("be liberal in what you accept and conservative in what you send")?

I hope I didn't say "Just use List" when we last talked...